<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Nutzerfenster</class>
 <widget class="QWidget" name="Nutzerfenster">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nutzerfenster</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../img/icon_Nutzer.png</normaloff>../img/icon_Nutzer.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>51</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../img/iFridge_klein.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Terminal&quot;;</string>
   </property>
   <property name="text">
    <string>Nutzer</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>351</width>
     <height>641</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="rightMargin">
     <number>0</number>
    </property>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_1">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	font: 75 9pt &quot;Terminal&quot;;
    border-radius: 45px; /* Setzen Sie den Radius auf die Hälfte der Breite und Höhe, um einen Kreis zu erstellen */
    background-color: #3498db; /* Farbe des Kreises */
    color: white; /* Farbe des Textes im Kreis */
    border: 5px solid #2980b9; /* Optional: Rand des Kreises */
    width: 80px; /* Breite des Kreises */
    height: 80px; /* Höhe des Kreises */
    font-size: 14px; /* Schriftgröße des Textes im Kreis */
}

QPushButton:pressed {
    background-color: #2980b9; /* Farbe des Kreises beim Drücken */
}</string>
      </property>
      <property name="text">
       <string>1: </string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	font: 75 9pt &quot;Terminal&quot;;
    border-radius: 45px; /* Setzen Sie den Radius auf die Hälfte der Breite und Höhe, um einen Kreis zu erstellen */
    background-color: #3498db; /* Farbe des Kreises */
    color: white; /* Farbe des Textes im Kreis */
    border: 5px solid #2980b9; /* Optional: Rand des Kreises */
    width: 80px; /* Breite des Kreises */
    height: 80px; /* Höhe des Kreises */
    font-size: 14px; /* Schriftgröße des Textes im Kreis */
}

QPushButton:pressed {
    background-color: #2980b9; /* Farbe des Kreises beim Drücken */
}</string>
      </property>
      <property name="text">
       <string>2: </string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton_3">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	font: 75 9pt &quot;Terminal&quot;;
    border-radius: 45px; /* Setzen Sie den Radius auf die Hälfte der Breite und Höhe, um einen Kreis zu erstellen */
    background-color: #3498db; /* Farbe des Kreises */
    color: white; /* Farbe des Textes im Kreis */
    border: 5px solid #2980b9; /* Optional: Rand des Kreises */
    width: 80px; /* Breite des Kreises */
    height: 80px; /* Höhe des Kreises */
    font-size: 14px; /* Schriftgröße des Textes im Kreis */
}

QPushButton:pressed {
    background-color: #2980b9; /* Farbe des Kreises beim Drücken */
}</string>
      </property>
      <property name="text">
       <string>3: </string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton_4">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	font: 75 9pt &quot;Terminal&quot;;
    border-radius: 45px; /* Setzen Sie den Radius auf die Hälfte der Breite und Höhe, um einen Kreis zu erstellen */
    background-color: #3498db; /* Farbe des Kreises */
    color: white; /* Farbe des Textes im Kreis */
    border: 5px solid #2980b9; /* Optional: Rand des Kreises */
    width: 80px; /* Breite des Kreises */
    height: 80px; /* Höhe des Kreises */
    font-size: 14px; /* Schriftgröße des Textes im Kreis */
}

QPushButton:pressed {
    background-color: #2980b9; /* Farbe des Kreises beim Drücken */
}</string>
      </property>
      <property name="text">
       <string>4: </string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pushButton_5">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	font: 75 9pt &quot;Terminal&quot;;
    border-radius: 45px; /* Setzen Sie den Radius auf die Hälfte der Breite und Höhe, um einen Kreis zu erstellen */
    background-color: #3498db; /* Farbe des Kreises */
    color: white; /* Farbe des Textes im Kreis */
    border: 5px solid #2980b9; /* Optional: Rand des Kreises */
    width: 80px; /* Breite des Kreises */
    height: 80px; /* Höhe des Kreises */
    font-size: 14px; /* Schriftgröße des Textes im Kreis */
}

QPushButton:pressed {
    background-color: #2980b9; /* Farbe des Kreises beim Drücken */
}</string>
      </property>
      <property name="text">
       <string>5: </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>40</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../img/icon_zuruck.png</normaloff>../img/icon_zuruck.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>140</y>
     <width>351</width>
     <height>641</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <property name="rightMargin">
     <number>0</number>
    </property>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_6">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	font: 75 9pt &quot;Terminal&quot;;
    border-radius: 45px; /* Setzen Sie den Radius auf die Hälfte der Breite und Höhe, um einen Kreis zu erstellen */
    background-color: #3498db; /* Farbe des Kreises */
    color: white; /* Farbe des Textes im Kreis */
    border: 5px solid #2980b9; /* Optional: Rand des Kreises */
    width: 80px; /* Breite des Kreises */
    height: 80px; /* Höhe des Kreises */
    font-size: 14px; /* Schriftgröße des Textes im Kreis */
}

QPushButton:pressed {
    background-color: #2980b9; /* Farbe des Kreises beim Drücken */
}</string>
      </property>
      <property name="text">
       <string>6: </string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton_7">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	font: 75 9pt &quot;Terminal&quot;;
    border-radius: 45px; /* Setzen Sie den Radius auf die Hälfte der Breite und Höhe, um einen Kreis zu erstellen */
    background-color: #3498db; /* Farbe des Kreises */
    color: white; /* Farbe des Textes im Kreis */
    border: 5px solid #2980b9; /* Optional: Rand des Kreises */
    width: 80px; /* Breite des Kreises */
    height: 80px; /* Höhe des Kreises */
    font-size: 14px; /* Schriftgröße des Textes im Kreis */
}

QPushButton:pressed {
    background-color: #2980b9; /* Farbe des Kreises beim Drücken */
}</string>
      </property>
      <property name="text">
       <string>7: </string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton_8">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	font: 75 9pt &quot;Terminal&quot;;
    border-radius: 45px; /* Setzen Sie den Radius auf die Hälfte der Breite und Höhe, um einen Kreis zu erstellen */
    background-color: #3498db; /* Farbe des Kreises */
    color: white; /* Farbe des Textes im Kreis */
    border: 5px solid #2980b9; /* Optional: Rand des Kreises */
    width: 80px; /* Breite des Kreises */
    height: 80px; /* Höhe des Kreises */
    font-size: 14px; /* Schriftgröße des Textes im Kreis */
}

QPushButton:pressed {
    background-color: #2980b9; /* Farbe des Kreises beim Drücken */
}</string>
      </property>
      <property name="text">
       <string>8: </string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton_9">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	font: 75 9pt &quot;Terminal&quot;;
    border-radius: 45px; /* Setzen Sie den Radius auf die Hälfte der Breite und Höhe, um einen Kreis zu erstellen */
    background-color: #3498db; /* Farbe des Kreises */
    color: white; /* Farbe des Textes im Kreis */
    border: 5px solid #2980b9; /* Optional: Rand des Kreises */
    width: 80px; /* Breite des Kreises */
    height: 80px; /* Höhe des Kreises */
    font-size: 14px; /* Schriftgröße des Textes im Kreis */
}

QPushButton:pressed {
    background-color: #2980b9; /* Farbe des Kreises beim Drücken */
}</string>
      </property>
      <property name="text">
       <string>9: </string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pushButton_10">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	font: 75 9pt &quot;Terminal&quot;;
    border-radius: 45px; /* Setzen Sie den Radius auf die Hälfte der Breite und Höhe, um einen Kreis zu erstellen */
    background-color: #3498db; /* Farbe des Kreises */
    color: white; /* Farbe des Textes im Kreis */
    border: 5px solid #2980b9; /* Optional: Rand des Kreises */
    width: 80px; /* Breite des Kreises */
    height: 80px; /* Höhe des Kreises */
    font-size: 14px; /* Schriftgröße des Textes im Kreis */
}

QPushButton:pressed {
    background-color: #2980b9; /* Farbe des Kreises beim Drücken */
}</string>
      </property>
      <property name="text">
       <string>10: </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
